appveyor-windows-build-rust.yml

```diff
@@ -0,0 +1,81 @@
+version: 1.0.{build}
```

Comment:
I think we need this since GH actions are not supporting some stuff for windows?

Reply:
yes, unfortunately GH Actions' Windows runner doesn't have the Docker support (switch to Linux containers) that we need. At least I couldn't figure out how to do that as of now.